proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceColumn {
```

Comment:
why not just use attributekey here?

Reply:
I wanted something more specific than `AttributeKey` since we don't have dynamic attributes support and we want some arbitrary fields like `ROOT_SPAN_NAME` or the different span counts.

We can't aggregate traces arbitrarily like we do for spans, so having a specific list of what we can support seemed more useful than making it completely dynamic.